app/src/main/java/org/luxoft/sdl_core/BleCentralService.java

```diff
@@ -102,6 +105,14 @@ public void run() {
                         mCallback = new BleAdapterWriteMessageCallback();
                         if (mNativeBleAdapterThread != null) {
                             mNativeBleAdapterThread.ReadMessageFromNative(mCallback);
+
+                            BluetoothPeripheral peripherial = mBluetoothHandler.getPeripheral();
```

Comment:
Maybe we can incapsulate this logic inside of mBluetoothHandler class? I'd prefer to leave BleCentralService class agnostic for used libraries for BLE handling, so I propose to get string ctrlMEssage from mBluetoothHandler and remove all references to com.welie.blessed.BluetoothPeripheral

Reply:
done in 66d56ca414ac2d17dea823a2186a8d3f2569ec83